src/Cooked/MockChain/BlockChain.hs

```diff
@@ -381,6 +382,33 @@ txSkelInputDataAsHashes skel = do
   (Map.elems -> inputTxOuts) <- txSkelInputUtxos skel
   catMaybes <$> mapM outputToDatumHashM inputTxOuts
 
+-- | This creates a payment from an existing TxOut. This is not trivial because
+-- we need to reconstruct a bunch of information using data withing the blockchain
+txOutRefToTxSkelOut :: (MonadBlockChainBalancing m) => Api.TxOutRef -> m TxSkelOut
+txOutRefToTxSkelOut oRef = do
```

Comment:
I think it might have really come in handy in a cooked GUI for interactive edition of UTxO post creation.

Out of curiosity, what is your use case for skeleton reconstruction?